models/tokenlon/ethereum/tokenlon_ethereum_schema.yml

```diff
@@ -0,0 +1,292 @@
+version: 2
+
+models:
+  - name: tokenlon_v5_ethereum_amm_v1_trades
```

Comment:
can you include a unique combo of columns test in the schema file here for all spells which are incremental? and the unique combo being the unique keys in the model?

there are examples in uniswap:
https://github.com/duneanalytics/spellbook/blob/main/models/uniswap/ethereum/uniswap_ethereum_schema.yml#L14